src/main/resources/assets/appliedenergistics2/lang/ja_JP.lang

```diff
@@ -50,6 +50,8 @@ tile.appliedenergistics2.BlockCraftingStorage4k.name=4k クラフティングス
 tile.appliedenergistics2.BlockCraftingStorage16k.name=16k クラフティングストレージ
 tile.appliedenergistics2.BlockCraftingStorage64k.name=64k クラフティングストレージ
 tile.appliedenergistics2.BlockCraftingAccelerator.name=MEクラフティングCPU
+tile.appliedenergistics2.BlockCraftingAccelerator4x.name=ME4芯クラフティングCPU
```

Comment:
You don't need to add other lang translations if you're not sure. (this is wrong translation)